public/app.js

```diff
@@ -0,0 +1,330 @@
+// AUTH CONTROLLER
+var authController = (function() {
```

Comment:
let?

Reply:
As discussed earlier this is going to be a class